src/nrtk_explorer/app/transforms.py

```diff
@@ -249,15 +278,14 @@ def on_hover(self, hover_event):
             self._on_hover_fn(id_)
 
     def settings_widget(self):
-        with html.Div(trame_server=self.server):
-            with html.Div(classes="col"):
-                self._parameters_app.transform_select_ui()
-
-                with html.Div(
-                    classes="q-pa-md q-ma-md",
-                    style="border-style: solid; border-width: thin; border-radius: 0.5rem; border-color: lightgray;",
-                ):
-                    self._parameters_app.transform_params_ui()
+        with html.Div(classes="col", trame_server=self.server):
```

Comment:
why do we provide trame_server? Is that not executed in the context of a layout?

Reply:
IDK, was following old code's suite.  Works just fine without.  Removed. ✅